Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -112,6 +119,21 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
     cell.contentView.alpha = 0.5
     #endif
   }
+    
+  func startDetermineLocation() {
+    if CLLocationManager.locationServicesEnabled() {
+      self.locationManager.startUpdatingLocation()
```

Comment:
It's like in the [UWP code snippet](https://docs.microsoft.com/en-us/appcenter/sdk/analytics/uwp#session-and-device-information) it's a one time thing. We don't need to register to all the location change. You can use the [requestLocation](https://developer.apple.com/documentation/corelocation/cllocationmanager/1620548-requestlocation) method instead of `startUpdatingLocation`.
Then you would rename the method `requestlocation` instead of `startDetermineLocation` :).

Reply:
When use requestlocation instead of startUpdatingLocation, need to implement the method didFailWithError